src/overlays/actors/ovl_Bg_Ddan_Jd/z_bg_ddan_jd.c

```diff
@@ -31,17 +32,152 @@ const ActorInit Bg_Ddan_Jd_InitVars = {
     (ActorFunc)BgDdanJd_Update,
     (ActorFunc)BgDdanJd_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Ddan_Jd/BgDdanJd_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Ddan_Jd/BgDdanJd_Destroy.s")
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
+};
+
+enum BgDdanJdState {
+    STATE_GO_BOTTOM,
+    STATE_GO_MIDDLE_FROM_BOTTOM,
+    STATE_GO_MIDDLE_FROM_TOP,
+    STATE_GO_TOP,
+};
```

Comment:
This is the style we typically use for enums:
```suggestion
typedef enum {
    STATE_GO_BOTTOM,
    STATE_GO_MIDDLE_FROM_BOTTOM,
    STATE_GO_MIDDLE_FROM_TOP,
    STATE_GO_TOP,
} BgDdanJdState;
```